test/io/test_io.py

```diff
@@ -320,6 +320,7 @@ def test_db_schema_reload(tmp_path, defaultenv):
         assert response.text == '"\\"v1\\", \\"public\\""'
 
 
+@pytest.mark.skip
 def test_db_schema_notify_reload(defaultenv):
```

Comment:
Somehow this test makes others fail, because of the execution of

https://github.com/PostgREST/postgrest/blob/973dca11bfe80566a44219f3b99bde0d6ec3aaee/test/io/fixtures.sql#L49-L56